haystack/pipelines/ray.py

```diff
@@ -61,14 +61,17 @@ class RayPipeline(Pipeline):
     YAML definitions of Ray pipelines are validated at load. For more information, see [YAML File Definitions](https://haystack-website-git-fork-fstau-dev-287-search-deepset-overnice.vercel.app/components/pipelines#yaml-file-definitions).
     """
 
-    def __init__(self, address: str = None, ray_args: Optional[Dict[str, Any]] = None):
+    def __init__(
+        self, address: str = None, ray_args: Optional[Dict[str, Any]] = None, serve_detached: Optional[bool] = False
+    ):
         """
         :param address: The IP address for the Ray cluster. If set to `None`, a local Ray instance is started.
         :param kwargs: Optional parameters for initializing Ray.
+        :param serve_detached: Optional parameter for initializing Ray Serve with the `detached` option.
```

Comment:
I would consider updating this docstring to `Whether or not to initialize Ray Serve with the "detached" option` since it is not per se an optional parameter.

Reply:
Hi @sjrl ,
You are right, the docstring has been fixed now as you have suggested.